tests/integration/security/testdata/requestauthn/e-authn.yaml.tmpl

```diff
@@ -10,7 +10,7 @@ spec:
       app: e
   jwtRules:
   - issuer: "test-issuer-1@istio.io"
-    jwksUri: "https://raw.githubusercontent.com/istio/istio/master/tests/common/jwt/jwks.json"
+    jwksUri: "http://jwt-server.{{ .Namespace }}.svc.cluster.local:8000/jwks"
```

Comment:
could you put this in a new policy and use a new workload? So that you won't need to touch the existing test and this makes sure we won't lose any coverage from the old tests.